src/main/java/ejercicio/banco/MainProcess.java

```diff
@@ -2,6 +2,7 @@
 import main.java.ejercicio.banco.dto.Account;
 import main.java.ejercicio.banco.dto.Bank;
 import main.java.ejercicio.banco.dto.Payment;
+import main.java.ejercicio.banco.repository.*;
```

Comment:
Try to avoid these wilcards when using the imports. This is not a good approach because you might be importing more classes than the needed.

https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports